scss/_orange-footer.scss

```diff
@@ -0,0 +1,209 @@
+.footer-mailing {
+  padding-top: $footer-orange-mailing-padding-top;
```

Comment:
looks like there's many micro-positionning based on different padding value on different footer type. Is there a way to have a `.footer` class that define default padding, maybe other styling also, and check if additionnal spacing could be handled by sub-component. Keep `.footer-*` classes sounds good for that purpose